tools/image_processing/imagej2/imagej2_convert_format/imagej2_convert_format.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_convert_format" name="Convert image format" version="1.0.0">
+    <description>with bioformats</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command interpreter="python">
+        imagej2_convert_format.py
+        -i $input
+        -d "${set_input_datatype.input_datatype}"
+        -t "${set_max_heap_size.max_heap_size_type}"
+        -m "${set_max_heap_size.max_heap_size}"
+        -q "${set_output_datatype.output_datatype}"
+        -o $output
+    </command>
+    <inputs>
+    	<expand macro="max_heap_size_type_conditional" />
+		<expand macro="input_datatype_conditional" />
+		<expand macro="output_datatype_conditional" />
+    </inputs>
+    <outputs>
+        <data name="output" format="jpg">
+            <change_format>
+            	<when input="set_output_datatype.output_datatype" value="jpg" format="jpg"/>
+                <when input="set_output_datatype.output_datatype" value="png" format="png"/>
+                <when input="set_output_datatype.output_datatype" value="tiff" format="tiff"/>
+            </change_format>
+        </data>
+    </outputs>
+  	<tests>
+      	<test>
+          	<param name="input" value="dot_blot.jpg" ftype="jpg" />
+          	<param name="max_heap_size_type" value="kilobytes" />
+          	<param name="max_heap_size" value="512" />
+          	<param name="input_datatype" value="jpg" />
+          	<output name="output_datatype" value="png" /> 
+          	<output name="output" file="dot_blot.png" ftype="png" />
+      	</test>
+      	<test>
+          	<param name="input" value="dot_blot.png" ftype="png" />
+          	<param name="max_heap_size_type" value="kilobytes" />
+          	<param name="max_heap_size" value="256" />
+          	<param name="input_datatype" value="png" />
+          	<output name="output_datatype" value="tiff" /> 
+          	<output name="output" file="dot_blot.tiff" ftype="tiff" />
+      	</test>
+      	<test>
+          	<param name="input" value="dot_blot.tiff" ftype="tiff" />
+          	<param name="input_datatype" value="tiff" />
+          	<output name="output_datatype" value="jpg" /> 
+          	<output name="output" file="dot_blot.jpg" ftype="jpg" />
+      	</test>
+  	</tests>
+    <help>
+**What it does**
```

Comment:
You should use CDATA where ever you can to avoid `&lt;` this.
I would recommend `planemo lint` to you, this can catch such not-best-practice stuff.